core/lib/spree/core/controller_helpers/current_host.rb

```diff
@@ -1,15 +0,0 @@
-# frozen_string_literal: true
-
-module Spree
-  module Core
-    module ControllerHelpers
-      module CurrentHost
```

Comment:
Deleting this module is a breaking change because people may have added it to their store's controllers. What do you think about deprecating it instead?

Reply:
Good call, @kennyadsl. Please, take another look.